app/build.gradle

```diff
@@ -3,6 +3,9 @@ apply plugin: 'com.android.application'
 android {
     compileSdkVersion 33
     buildToolsVersion "30.0.3"
+    packagingOptions {
+        resources.excludes.add("META-INF/*")
```

Comment:
After researching some more, it seems like the reason for the problematic META-INF files is because the google apis normally use org.apache.httpcomponents under the hood, which has a httpclient class that conflicts with androids own httpclient class (I'm very over my head so this could be wrong). When i take out the exclusion line, the error I get 
complains about META-INF files at jetified-httpclient-4.5.13.jar and jetified-httpcore-4.4.15.jar.

Although the exclusion option above seems to be something that people do or advice most commonly, the below link on how to set up google workspace API's for android includes a line below each dependency to exclude org.apache.httpcomponents. 

https://developers.google.com/workspace/guides/android

However, this link seems to use a much older version of gradle, and I was not able to make these exclusion statements work when mirroring their approach exactly. It seems like the internet's solution to this is to exclude org.apache.httpcomponents from all configurations 

configurations.all() {
    exclude group: "org.apache.httpcomponents"
}

It doesn't seem like excluding apache httpcomponents from everything should be a problem as I don't think we should be trying to use it for anything, but regardless I don't think I can get much farther online without advice from someone more knowledgeable, so I think I am going to go with excluding apache httpcomponents for now unless you have other advice. (I am thus removing the current META-INF exclusion).

Reply:
Thanks for investigating this. It looks like either way is fine, but this is new territory for me as well.